BigBaseV2/src/fonts/icons.sfd

```diff
@@ -0,0 +1,309 @@
+SplineFontDB: 3.2
+FontName: Untitled1
+FullName: Untitled1
+FamilyName: Untitled1
+Weight: Regular
+Copyright: Copyright (c) 2022, Tom Chan
```

Comment:
@GeopJr this seems like license problem to me, what do you think?

Reply:
I'll reply in review since it involves more than that file.